apps/schools/domains/user/components/auth/registerForm/index.tsx

```diff
@@ -28,12 +30,36 @@ export const RegisterForm: React.FC<IRegisterFormProps> = ({ onFinish, onError }
         signInByPhone: () => {},
     }
 
+    const [updateProfile] = useUpdateEmployeeProfileByIdMutation();
+    const { user } = useUserProfile()
+    const { data: allEmployees, isLoading: employeesLoading } = useGetAllEmployeesQuery({
+        employee_profile: user.employee_profile?.id,
+    });
+
     const registerComplete = useCallback(async () => {
-        const { password } = form.getFieldsValue(['password'])
-        registrationHandler(phone, password, userRegistration, onFinish, onError, form)
-    }, [form, signInByPhone])
+        const { password } = form.getFieldsValue(['password']);
+
+        await registrationHandler(phone, password, userRegistration, onFinish, onError, form)
+            .then(() => {
+                const { email } = form.getFieldsValue(['email']);
+                const { name } = form.getFieldsValue(['name']);
+
+                if (!employeesLoading && allEmployees) {
+                    if (user.employee_profile?.id) {
+                        const updateEmail = {
+                            employee_profile_id: user.employee_profile.id,
+                            name: name,
+                            email: email
+                        }
+                        updateProfile(updateEmail);
+                    }
+                }
```

Comment:
you can locate this code in the same directory as where registrationHandler stores